src/shared/logs-show.c

```diff
@@ -942,6 +942,19 @@ struct json_data {
         JsonVariant* values[];
 };
 
+static struct json_data* json_data_free(struct json_data *d) {
+        json_variant_unref(d->name);
```

Comment:
assert(d)

Reply:
or maybe better return NULL if d is NULL

Reply:
Ouch